tests/generate_code.sh

```diff
@@ -26,6 +26,7 @@ set -e
 ../flatc --cpp --java --kotlin --csharp --python --gen-mutable --reflect-names --gen-object-api --gen-compare --no-includes --cpp-ptr-type flatbuffers::unique_ptr monster_extra.fbs monsterdata_extra.json
 ../flatc --cpp --java --csharp --gen-mutable --reflect-names --gen-object-api --gen-compare --no-includes --scoped-enums --jsonschema --cpp-ptr-type flatbuffers::unique_ptr arrays_test.fbs
 ../flatc --python --gen-mutable --reflect-names --gen-object-api --gen-compare --scoped-enums --jsonschema --cpp-ptr-type flatbuffers::unique_ptr arrays_test.fbs
+../flatc --rust rust/required_fields_should_not_be_option.fbs
```

Comment:
This schema isn't necessarily Rust specific, is it?

Reply:
You're right. But, I have no idea which languages have to be generated. So, I didn't touch this. Can you give a suggestion about this? I'll follow yours.